lib/plugins/filter/before_post_render/indented_code_block.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const { escapeHTML, stripIndent } = require('hexo-util');
+
+const rIndentedCodeBlock = /(^|(?:^|\n)(?:(?: {0,3}>){1,3}[^\S\r\n])?[^\S\r\n]*\n)((?:((?:(?: {0,3}>){1,3}[^\S\r\n])?)( {4}|\t)([^\n]*?)(\n|$))+)/g;
+
+function indentedCodeBlock(data) {
+  data.content = data.content.replace(rIndentedCodeBlock, ($0, before, content, start, $4, $5, end) => {
```

Comment:
```suggestion
  # Return early if there is no 4 consecutive spaces
  if (!data.content.includes('    ')) return;
  data.content = data.content.replace(rIndentedCodeBlock, ($0, before, content, start, $4, $5, end) => {
```

Reply:
To support single-tab-indented code block, this optimization can not be used.